src/styles/header.scss

```diff
@@ -0,0 +1,81 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './stars';
+@import './colors';
+@import './card';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+
+  --header-height: 60px;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  width: 100%;
+  height: var(--header-height);
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
Consider using a variable for font-weight if you have a predefined set of font-weight values. This can help maintain consistency and make it easier to change the values in one place if needed.